### ISSUE TITLE
Adding support for SavedModel for converting model to tfjs_graph_model

### ISSUE THREAD

**Issue by binary-person** [ArunMichaelDsouza/tensorflow-image-detection#12]
The title says it all. Do you have time to make this enhancement?

**Comment by ArunMichaelDsouza** [comment]
Hi @scheng123, can you please explain what this enhancement would do? Also, I would really appreciate a PR if you have the time. 🙂

**Comment by binary-person** [comment]
Since I’m experienced in web development, I was planning to have the program export it’s own JavaScript model, so I can later on  incorporate that into a browser, which would be cool! And also, since I’m new to python, I have no idea how to do that (tried everything), although it would be awesome if I could make a PR. Do you have the time? I always love learning new things. 😊

**Comment by binary-person** [comment]
Looks like its impossible since the inception model was using a deprecated BatchNormWithGlobalNormalization, and an unsupported DecodeJpeg. Nice try though. Anyway, apart from the fact that converting it to a javascript model won't work anyway, you want me to make a pull request for adding support for the checkpoint and SavedModel saving feature?

**Comment by ArunMichaelDsouza** [comment]
Yes ofcourse, that would be great!

**Comment by binary-person** [comment]
Added checkpoint, and SavedModel saving feature. Also added a CTRL+C save as an extra bonus. As my use of language differs from yours, can you update the README.md to update the ability to save with the combination of CTRL+C? As always, I love to help out!
Check the code before merging, because I'm terrible at python.
Pull request: #14

**Comment by scheng123** [comment]
14#

**Comment by ArunMichaelDsouza** [comment]
Thanks a lot @scheng123! Please view https://github.com/ArunMichaelDsouza/tensorflow-image-detection/pull/14#issuecomment-488709802
